include/itemutils.h

```diff
@@ -0,0 +1,24 @@
+#ifndef NEWSBOAT_ITEMUTILS_H_
+#define NEWSBOAT_ITEMUTILS_H_
+
+#include <memory>
+
+#include "cache.h"
+#include "rssfeed.h"
+#include "rssitem.h"
+#include "view.h"
+
+namespace newsboat {
+
+/// Adds enclosure URL of item to queue.
+///
+/// Reports outcome on the statusline.
+///
+/// Returns true if URL was successfully added to queue or was already queued before,
+///         false otherwise.
+bool EnqueueItemEnclosure(std::shared_ptr<RssItem> item, std::shared_ptr<RssFeed> feed,
```

Comment:
Please change the function name to camel case, as that's the common style; see e.g. include/utils.h. I'd also be grateful if you could update doc/code-style.markdown on this point, because it only talks of "method names".

Reply:
Done.

I noticed that we have several namespaces which are using CamelCase instead of snake_case.
I changed those as well. Feel free to drop the extra renaming commits if you don't want those in this PR (or at all)

Reply:
Oh, that's a good idea! But then the files should be renamed to match: "filesystembrowser.*" should become "file_system.*", and "test-helpers" directory should become "test_helpers".

I swear this is my last ask for this PR :rofl:

Reply:
All done :)

Reply:
Thanks!